xpack_info.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2005-2018 Coveo Solution. All rights reserved.
```

Comment:
IANAL but this might probably be a problem. I don't want to run into any problems due to legal issues; it's just not worth it.

Reply:
I'm looking with our legal dep about what I can do. If they are alright with it I'll just change it back to whatever is okay with everyone.